### ISSUE TITLE
Add a CONTRIBUTING guide

### ISSUE THREAD

**Issue by raybesiga** [sparkplug/momoapi-python#19]
Need to add a contributing guide

**Comment by nanjekyejoannah** [comment]
Do you plan to include details on developer setup in the contribution guide?

Like how to setup for development.

**Comment by raybesiga** [comment]
Hi @nanjekyejoannah have you read through the README?

**Comment by nanjekyejoannah** [comment]
@raybesiga yes I have. The setup for development is a section with details like. get the code ..run tests..run one test etc

You have a section for running tests. The readme IMHO looks more like to a user of the package not a developer/contributor. 

I was also thinking, a docs folder may be? most projects have a readme and separate documentation. Again this decision is made by maintainer and w just follow.

**Comment by raybesiga** [comment]
@nanjekyejoannah appreciate your feedback. As you can see, we only just opened this issue 6 days ago. We shall work on a contributing guide and you will be the first to know. We look forward to your contributions! Best regards.

**Comment by raybesiga** [comment]
Hi @nanjekyejoannah please find a link to the update contributing guide [here](https://github.com/sparkplug/momoapi-python/blob/master/CONTRIBUTING.md). Thank you for your patience.

**Comment by nanjekyejoannah** [comment]
Thanks @raybesiga . Its  what I envisioned. I can close this I guess. Also I will find time to contribute to some work on this project. If you have any burning open issues I can help with...do tell.

**Comment by raybesiga** [comment]
That is great to hear @nanjekyejoannah. We may have blind spots and could use contributions both for bugs, as well as enhancements and feature requests. Open a ticket in case anything comes to mind.
